src/components/CreateArticleForm.jsx

```diff
@@ -34,9 +34,11 @@ const CreateArticleForm = ({ onCreateMessage }) => {
     event.preventDefault();
     const response = await Articles.create(article);
     onCreateMessage(response);
-    setArticle(InitialArticle);
-    setFileName("");
-    setTimeout(() => onCreateMessage(""), 4000);
+    if (response === "Article succesfully created!") {
```

Comment:
the message must match the one we get from the api, so che conditional should be:
` if (response === "Article created successfully") {`